less/cards.less

```diff
@@ -456,4 +456,47 @@
 
 .card-small {
   max-width: 30px;
+}
+
+.drag-preview {
+  pointer-events: none;
+  position: fixed;
+
+  opacity: 0.5;
+  z-index: 9999;
+}
+
+.relative {
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  padding: inherit;
+  margin: inherit;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 9999;
```

Comment:
May be worth pulling this into a z-index variable in variables.less. 9999 is already defined as `@layer-top`, probably worth looking at what else is using that and seeing if the drag interactions need to be even higher.

Reply:
right, knew there was something i was supposed to go back and change..